packages/shared/components/modals/NetworkSummaryModal.svelte

```diff
@@ -3,43 +3,51 @@
     import {
         NETWORK_HEALTH_COLORS,
         networkStatus,
-        NetworkStatusDescription,
+        NETWORK_STATUS_DESCRIPTION,
         NetworkHealth,
         nodeInfo,
     } from '@core/network'
     import { activeProfile } from '@core/profile'
     import { localize } from '@core/i18n'
+    import { TextType } from 'shared/components/Text.svelte'
 
     export let modal: Modal
 
-    $: health = $networkStatus.health ?? 0
-    $: description = $networkStatus.description ?? NetworkStatusDescription[NetworkHealth.Disconnected]
-    $: messagesPerSecond = $networkStatus.messagesPerSecond ?? 0
-    $: referencedRate = $networkStatus.referencedRate ?? 0
+    let health: NetworkHealth
+
+    $: health = $networkStatus.health ?? NetworkHealth.Down
+    $: healthColor = `${NETWORK_HEALTH_COLORS[health]}-500`
+    $: description = $networkStatus.description ?? NETWORK_STATUS_DESCRIPTION[NetworkHealth.Disconnected]
+    $: networkStatistics = {
+        messagesPerSecond: Math.round($networkStatus.messagesPerSecond ?? 0),
+        referencedRate: Math.round($networkStatus.referencedRate ?? 0) + '%',
+    }
 </script>
 
 <Modal bind:this={modal} position={{ left: '80px', top: '25px' }}>
     <network-indicator-content class="flex flex-col">
-        {#if $activeProfile?.isDeveloperProfile}
-            <Text type="h3" classes="px-7 pt-5">{$nodeInfo?.protocol?.networkName}</Text>
-        {/if}
-        <div class="px-7 pb-5 text-13 text-{NETWORK_HEALTH_COLORS[health]}-500">
-            {localize(`views.dashboard.network.${description}`)}
+        <div class="py-5">
+            {#if $activeProfile?.isDeveloperProfile}
+                <Text type={TextType.h3} classes="px-7">{$nodeInfo?.protocol?.networkName}</Text>
+            {/if}
```

Comment:
```suggestion
            {#if $activeProfile?.isDeveloperProfile && $nodeInfo?.protocol?.networkName}
                <Text type={TextType.h3} classes="px-7">{$nodeInfo?.protocol?.networkName}</Text>
            {/if}
```

if any of `nodeinfo.protocol.networkname` can be undefined, add it to the if condition, else an empty Text element will be rendered. 

If it cannot be undefined, remove the `?`